packages/cyberstorm/src/components/Layout/PackageDetailLayout/PackageManagementForm/PackageManagementForm.tsx

```diff
@@ -4,13 +4,16 @@ import { TextInput } from "../../../TextInput/TextInput";
 import { Alert } from "../../../Alert/Alert";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faCircleExclamation } from "@fortawesome/pro-solid-svg-icons";
+import { Button } from "../../../..";
+import { Package } from "@thunderstore/dapper/types";
 
 export interface PackageManagementFormProps {
   packageStatus?: string;
+  packageData: Package;
```

Comment:
Dunno if you're noticed while reviewing my changes, but I've actively tried to avoid the `packageData` as a prop name since it's clunky and redundant. Anyway, in this case it would seem that simply passing `isDeprecated` as a prop would suffice.